_data/languages.yml

```diff
@@ -1,30 +1,1065 @@
---- 
+---
+-
+  code: af
+  name: Afrikaans
+  family: ine
+  scripts: [ Latn ]
+  typology:
+    - SVO, VSO
+    - analytic
+  variants: [ za ]
+
+-
+  code: ar
+  name: Arabic
+  family: sem
+  scripts: [ Arab, Syrc ]
+  typology:
+    - VSO
+    - fusional
+  variants: [ dz, bh, eg, iq, jo, kw, lb, ly, ma, om, qa, sa, sy, tn, ae, ye, sa ]
+
+-
+  code: az
+  name: Azerbaijani
+  family: trk
+  scripts: [ Latn, Cyrl ]
```

Comment:
The majority of Azeri Turkish speakers live in Iran, where the Arabic script is also used, and that was also the main script for Azeri Turkish for most of its history.